content/docs/reference/spec/buildpack-api.md

```diff
@@ -184,9 +189,31 @@ The schema is as follows:
         - **`uri`** _(string, optional)_\
         A URL or path to the license.
 
-- **`stacks`** _(list, optional)_\
+- **`targets`** _(list, optional)_\
+  A list of targets supported by the buildpack.
+  _Cannot be used in conjunction with `order` list._
+
+  - **`os`** _(string, optional)_\
+    The supported operating system name.
+
+  - **`arch`** _(string, optional)_\
```

Comment:
I am a little confused, **when** `targets` are used, all the attributes are optional? `os` must be mandatory right?

Reply:
In the end we decided that even `os` could be optional - though practically speaking, it will probably always be specified. I added some clarifying language here: https://github.com/buildpacks/docs/pull/618/files#r1334413773